### ISSUE TITLE
Issue merge request to portswigger repo

### ISSUE THREAD

**Issue by ngo** [nccgroup/Decoder-Improved#10]
BApp Store version of decoder-Improved is quite old. In order to update it you should issue a pull-request to https://github.com/PortSwigger/decoder-improved repo. Would you consider doing that?

**Comment by ngo** [comment]
I took liberty of issuing the PR: https://github.com/PortSwigger/decoder-improved/pull/1

**Comment by justinmoore** [comment]
hey @ngo,

Sorry for the delay on this, I've been on vacation and focusing on my other plugin. I appreciate you taking the initiative to get this updated on the BApp store.

The way I've been updating my plugins is by emailing the PortSwigger folks directly and having them do it. I'll merge your PR and email them to update the plugin.

Thanks for helping out,
Justin

**Comment by ngo** [comment]
Actually, guys at PortSwigger do follow through on pull requests (with a couple days' delay), which I found to be more convenient that mailing them (See e.g. https://github.com/PortSwigger/request-minimizer/pulls?q=is%3Apr+is%3Aclosed). The only thing I had to change in my workflow was to merge back changes they made (i.e. manifest files) and remember to bump version on each PR, so that the commit history remains nicely in sync.
